scrapers/parse_scrape_output.py

```diff
@@ -24,17 +24,17 @@
 
 months_fr = {
   "janvier": 1,
-  "fèvrier": 2,
+  "février": 2,
```

Comment:
Since we use this for parsing, let's just have both variants in the dict. So if it's different on the website, we can still parse it correctly.

Reply:
Updated all 3